conman/routes/tests/test_models.py

```diff
@@ -45,6 +49,16 @@ def test_unique_url(self):
             Route.objects.create(url='/')
 
 
+class RouteCheckTest(TestCase):
+    """Test Route.check()."""
+    def test_defer_to_handler(self):
+        """Route.check returns the result of Route.handler_class().check()."""
+        with mock.patch('conman.routes.models.Route.handler_class') as handler:
+            errors = Route.check()
+
+        self.assertEqual(errors, handler.check(Route))
```

Comment:
Aha, ok.

Well in that case, arguably, yes, it could be better in some ways. In particular, it would be a single assertion in a single line, where mine is sort of a 2-for-1.

If yours addresses the fact that `check` is called a single time with `Route` as the argument, perhaps I should add another assertion about the return value?